tests/testproject/TestProject.AppHost/Directory.Build.props

```diff
@@ -3,6 +3,6 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
   <!-- NOTE: This line is only required because we are using P2P references, not NuGet. It will not exist in real apps. -->
-  <Import Project="..\..\..\src\Aspire.Hosting\build\Aspire.Hosting.props" Condition="'$(TestsRunningOutsideOfRepo)' != 'true'" />
+  <Import Project="../../../src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.props" Condition="'$(TestsRunningOutsideOfRepo)' != 'true'" />
```

Comment:
There is a mix of forward and black slashes in project files.

Reply:
Yeah, I wasn't going to address that here. But I can.

The majority of .props all have forward slashes. The majority of the .targets have backslashes.